blocks/carousel/carousel.css

```diff
@@ -263,4 +263,15 @@ main .section div.carousel-wrapper {
         text-align: left;
         margin: 0;
     }
+}
+
+@media screen and (max-width: 900px) {
```

Comment:
Let's not create a new breakpoint this way. Update the values (e.g. line 81 margin) to 32px, then put the 24px back in with the existing desktop breakpoint (after line 195).